bears/js/ESLintBear.py

```diff
@@ -47,7 +47,12 @@ def create_arguments(filename, file, config_file,
 
     @staticmethod
     def generate_config(filename, file):
-        return '{"extends": "eslint:recommended"}'
+        return """
+        {
+            "extends": "eslint:recommended",
+            "parser": "babel-eslint",
```

Comment:
It *is* the most common practice, but I am also very hesitant on making it default. I'll revert it during the rebasing just to keep stuff simple.